### ISSUE TITLE
hope to refer to pure pytorch Version of the source code

### ISSUE THREAD

**Issue by westream** [CompVis/iin#1]
Hello, we are preparing for the work of cvpr2020, I hope you can quote your work, but the edflow library you use, although it runs very well, its debugging and use is still very difficult for me, I still hope to refer to pure pytorch Version of the source code

**Comment by pesser** [comment]
Hi,
Since we are using edflow extensively for our projects, we cannot provide a standalone version. However, if you are encountering a specific problem with it, we'd be happy to help! In any case, edflow mostly handles the necessary logistics of logging, parsing configs, loading and iterating over batches of data etc. The `model` and `loss` classes can be used independent of edflow. You would then only need to write your own dataloader and adapt the logic of the `iterator` class in your own loop over the batches.
Best
